mod_reforged/config/item_names.nut

```diff
@@ -1,3 +1,52 @@
+// Vanilla Adjustments
+::MSU.Array.removeValues(::Const.Strings.CrossbowNames, [
+	"Betrayer",		// moved over to dagger names
+]);
+::Const.Strings.CrossbowNames.extend([
+	"Scorpion",
+	"Ironsight",
+	"Windlass",
+]);
+
+::MSU.Array.removeValues(::Const.Strings.DaggerNames, [
+	"Estoc",		// We have our own actual Kriegsmesser item
+]);
+::Const.Strings.DaggerNames.extend([
+	"Betrayer",		// formerly a crossbow name
+]);
+
+::Const.Strings.HandgonneNames.extend([
+	"Deafener",
+	"Eisenknall",
+	"Volcano",
+]);
+
+::Const.Strings.LongaxeNames.extend([
+	"Culler",
+]);
+
+::Const.Strings.SwordlanceNames.extend([	// In Vanilla this has only 2 entries
+	"Bloodmoon",
+	"Ghoul\'s Bane",
+	"Soulstrike",
+	"Shadowreaper",
+	"Death\'s Hand",
+	"Necroblade",
+	"Grimharvest",
+]);
+
+::MSU.Array.removeValues(::Const.Strings.WarbrandNames, [
+	"Kriegsmesser",	// We have our own actual Kriegsmesser item
+]);
+
+::Const.Strings.WhipNames.extend([
```

Comment:
Other cool whip names:
- Spikelash
- Thornwhip
- Bloodlash
- Ironfang
- Ripper
- Warthorn

Reply:
- Yeah Ironfang isn't the best for a whip I guess.
- Ripper could be `Fleshripper`.
- Warthorn sounds ok to me. What do you find wrong about it?

Reply:
`War` is very generic
And `Thorn` only makes sense for the barbarian `thorned_whip` which does not have a famed version. I don't see how it fits with normal whips. That term would be better suited for spiked maces, pickaxes, spears, daggers.

Reply:
I mean to me it sounds like this is a whip which will prick you like a thorn and it's used in war. So it's a war thorn in the sense that it pricks people and makes them bleed. But no big deal you can ignore the name.

Reply:
Fleshripper is already used for Kriegsmesser so I will not add it here